html/pfappserver/root/static.alt/src/views/Configuration/_config/provisioning.js

```diff
@@ -834,6 +834,20 @@ export const viewFields = {
       ]
     }
   },
+  ca_cert_path: (form = {}, meta = {}) => {
+    return {
+      label: i18n.t('RADIUS server CA path'),
+      text: i18n.t('The path to the RADIUS server CA which have sign the radius certification.'),
```

Comment:
The path to the RADIUS server CA which signed the server certificate

Reply:
You still have bad verbiage (i.e. `which have sign the radius certification` is bad)

Reply:
Where? Latest commit did not fix it?

Reply:
Nope still says 'which signed the radius certification'

Reply:
Oh that is right... I missed the point.